### PR TITLE
Replace copyright notice in license headers and add AUTHORS.md

### PR DESCRIPTION
**Pull request by ubruhin**
@The-Compiler, @NiFa18, @chrisgwerder, @0xB767B:
I have added your GitHub usernames to the AUTHORS.md. Please contact me if you wish to be removed from the list, or if I should add your real name.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Fixes #62
 - Make sure you have added the necessary tests for your changes.
 - Run all tests to ensure nothing else was accidentally broken.
   - This is done by running the binary `./generated/unix|mac|windows/tests`
+- If you like, feel free to add yourself to the [AUTHORS.md](https://github.com/LibrePCB/LibrePCB/blob/master/AUTHORS.md) file.
```
Comment (rnestler) [inline CONTRIBUTING.md]: A bit too long this line maybe?

File: AUTHORS.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Many people have contributed to LibrePCB. This is an attempt to list them all.
+
+## Main Authors
+- U. Bruhin (ubruhin)
+
+## Contributors
+- Raphael Nestler (rnestler)
+- Danilo Bargen (dbrgn)
+- The-Compiler
```
Comment (The-Compiler) [inline AUTHORS.md]: I'd prefer "Florian Bruhin (The-Compiler)" :wink:

File: CONTRIBUTING.md (hunk 1 above)
Comment (ubruhin) [inline CONTRIBUTING.md]: Only this line? xD

File: AUTHORS.md (hunk 2 above)
Comment (ubruhin) [inline AUTHORS.md]: Done!

**Comment by dbrgn** [comment]
I would prefix the Github usernames with @, even if the usernames aren't linked automatically :)

**Comment by ubruhin** [comment]
> I would prefix the Github usernames with @, even if the usernames aren't linked automatically :)

I have now also added hyperlinks to all usernames ;)

**Comment by dbrgn** [comment]
Hypa hypa! :D

**Pull request merged**